src/hipscat/catalog/pixel_tree.py

```diff
@@ -2,7 +2,7 @@
 
 import pandas as pd
 
-from hipscat.catalog import PixelNode, PixelNodeType
+from hipscat.catalog import Catalog, PixelNode, PixelNodeType
```

Comment:
I was thinking eventually (though hopefully I'll get around to it before too long) the Catalog class would use the pixel tree to do filtering. Could we keep these constants elsewhere?

Reply:
Elsewhere done.